sdk/python/feast/infra/online_stores/contrib/postgres.py

```diff
@@ -95,6 +95,18 @@ async def _get_conn_async(
                 self._conn_async = await _get_conn_async(config.online_store)
             yield self._conn_async
 
+    async def close(self):
```

Comment:
Can you add a test for this?

Reply:
I'm still in a small discussion on the [issue](https://github.com/feast-dev/feast/issues/4401) linked to this PR. Once we agreed there I'll make sure to add a test!